docs-v2/content/en/docs/pipeline-stages/renderers/helm.md

```diff
@@ -22,7 +22,8 @@ Skaffold supports projects set up to render and/or deploy with Helm, but certain
 Skaffold no longer requires the intricate configuring of `artifactOverrides` or `imageStrategy` fields. See docs [here]({{< relref "#image-reference-strategies" >}}) on how old `artifactOverrides` and `imageStrategy` values translate to `setValues` entires in the latest Skaffold schemas (`apiVersion: skaffold/v3alpha1` or skaffold binary version `v2.0.0` onwards)
 {{< /alert >}}
 
-{{< alert title="In Skaffold `v2` the primary difference between the helm renderer (`manifest.helm.*`) and the helm deployer `deploy.helm.*` is the use of `helm template` vs `helm install`" >}}
+{{< alert title="Note" >}}
+In Skaffold `v2` the primary difference between the helm renderer (`manifest.helm.*`) and the helm deployer (`deploy.helm.*`) is the use of `helm install` vs `helm install
```

Comment:
should this be `helm template` vs `helm install` ?

Reply:
thanks for catching this, this was not intentional. Fixed now (to what you have above)